podman/api/ssh.py

```diff
@@ -202,6 +203,7 @@ def connect(self) -> None:
 class SSHConnectionPool(urllib3.HTTPConnectionPool):
     """Specialized HTTPConnectionPool for holding SSH connections."""
 
+    # pylint: disable=invalid-name
     ConnectionCls = SSHConnection
```

Comment:
```suggestion
    ConnectionCls = SSHConnection     # pylint: disable=invalid-name
```
Should disable the error for just this line, not the whole file.

Reply:
Fixed